src/hooks/useDebounce.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+
+function useDebounce(value, delay) {
```

Comment:
This hooks is used in the autocompleted to not do a fetch everytime the search is updated?

Reply:
yes to wait for the delay before sending the request, if no text change in that delay time it will perform the action